todo_app/templates/index.html

```diff
@@ -11,8 +11,16 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for todo in todos %}
+          <li>{{todo.title}}</li>
+        {% endfor %}
       </ul>
+      <h2>Update your To-Do List</h2>
+      <form method="POST" action="/add-todo">
+          <label>Add a new item</label>
+          <input type="text" name="todo" value=""></input>
```

Comment:
Minor, but adding a `required` attribute just to take advantage of the browser-side validation could be a nice simple addition here 👍